components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -586,14 +586,12 @@ var nodeProps = `
 var NodeState = Mapping{
 	Index:      "node-state-7",
 	Alias:      "node-state",
-	Type:       "node-state",
```

Comment:
As per my understanding, this is the custom type, and not the internal type. Do we need to remove these types as well?

Reply:
it is not needed in opensearch